examples/examples-cpu/nyc-taxi-snowflake/hyperparameter-dask.ipynb

```diff
@@ -186,7 +186,7 @@
    "cell_type": "markdown",
```

Comment:
<p>is there a reason to have <code>query</code> inside the function? I kind of like it outside because it generalizes the <code>load()</code> function. Users could copy/paste it into their own notebooks if they wanted. It also sorta matches how queries are used in the single-node manner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
no particular reason. I just thought it looked cleaner this way, but I'll switch it back